tests/helpers.mjs

```diff
@@ -0,0 +1,63 @@
+import assert from 'node:assert';
+import { join } from 'node:path';
+
+import { beforeAll, afterAll } from 'vitest';
+import tmp from 'tmp-promise';
+import { execa } from 'execa';
+import copyWithTemplate from '../lib/copy-with-template';
+
+const blueprintPath = join(__dirname, '..');
+const appName = 'fancy-app-in-test';
+
+export function newProjectWithFixtures({
+  flags = [],
+  fixturePath,
+  name = appName,
+} = {}) {
+  let tmpDir;
+  let emberCli = join(__dirname, '../node_modules/ember-cli/bin/ember');
```

Comment:
separate PR for this change: https://github.com/embroider-build/app-blueprint/pull/88